pkg/apis/operator/v1alpha1/tektonhub_types.go

```diff
@@ -63,6 +64,13 @@ type ApiSpec struct {
 	RouteHostUrl  string `json:"routeHostUrl,omitempty"`
 }
 
+type UiSpec struct {
+	ApiUrl      string `json:"apiUrl,omitempty"`
```

Comment:
can user provide api url?
and operator will use that instead of api it installed?

Reply:
For now, no user can't provide an api url, 
but in future we can add a mechanism for this